src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -31,14 +35,20 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = ({ isOpen, isLoading }) =
   const bestBuyAmountCryptoPrecisionAfterSlippage = bnOrZero(bestBuyAmountCryptoPrecision)
     .times(bn(1).minus(bnOrZero(bestQuote?.recommendedSlippage ?? DEFAULT_SLIPPAGE)))
     .toString()
-  const bestBuyAssetTradeFeeCryptoPrecision =
-    buyAssetFiatRate && bestQuote
-      ? bnOrZero(bestQuote.feeData.buyAssetTradeFeeUsd).div(buyAssetFiatRate)
-      : undefined
-  const bestTotalReceiveAmountCryptoPrecision = bestBuyAssetTradeFeeCryptoPrecision
+
+  const bestTotalProtocolFeeCryptoPrecision =
+    bestQuote !== undefined
+      ? sumProtocolFeesToDenom({
+          cryptoMarketDataById,
+          protocolFees: bestQuote.feeData.protocolFees,
+          outputExponent: buyAsset.precision,
+          outputAssetPriceUsd: buyAssetUsdRate,
+        })
+      : '0'
+
+  const bestTotalReceiveAmountCryptoPrecision = bestQuote
     ? bnOrZero(bestBuyAmountCryptoPrecisionAfterSlippage)
-        // TODO: determine why not subtracting bestSelllAssetTradeFeeCryptoPrecision
-        .minus(bestBuyAssetTradeFeeCryptoPrecision)
+        .minus(bestTotalProtocolFeeCryptoPrecision)
```

Comment:
pretty sure this is also a production bug. This will affect `quoteDifference`, the little red % in the UI that makes no sense (because the number is nonsense currently)